CMakeLists.txt

```diff
@@ -121,6 +121,10 @@ option (USE_STATIC_OSL                      "Use static OpenShadingLanguage libr
 option (WARNINGS_AS_ERRORS                  "Treat compiler warnings as errors"                     ON)
 option (HIDE_SYMBOLS                        "When using gcc, hide symbols not on the public API"    ON)
 
+if (CMAKE_SYSTEM_NAME MATCHES "Linux|FreeBSD")
+    option (USE_VISIBILITY_MAP              "When using gcc, hide symbols not on the public API"    OFF)
```

Comment:
It happens after. It can be used to hide more symbols.
The visibility map included hides OSL and boost symbols that are still on the library for example.
I'm still experimenting with this, that's why by default it is off.